README.md

```diff
@@ -2,12 +2,19 @@
 
 Replace this readme with your own information about your project.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This week we practiced on fetching API from github by extracting data from our forked Technigo repos. 
+We also learned how to use tokens and adding a chart with chart.js.
 
 ## The problem
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
+This time I learned to ask for help sooner than later since I got stuck with fetching pull requests inside a function. 
+The problem was solved by nesting the fetch of API for pull requests inside the first fetch. Another problem I encountered was not knowing how to use the tokens correctly and how to make Netlify show my index.html site. But after some added information in the notion page the problem was easily fixed! 
+If I had more time I would try to play around with API:s more and presenting the data with another chart.
+
```

Comment:
Interesting and detailed description of problem solving.

Reply:
Thank you so much for all your inputs, Josse. I am glad you found the comments to the code easy to follow too! :)